admin/src/athletes/Edit.vue

```diff
@@ -390,7 +387,7 @@ export default {
           if (res.body.length === 1) {
             this.currentItem = {
               ...res.body[0],
-              newMembership: []
+              newMembership: this.clubs.filter(club => club.thorId === this.currentItem.club)
```

Comment:
Very nice

Reply:
Hmm does this work locally for you?
I think `this.currentItem` is not set at this moment so you would have to do: `res.body[0].club)`

Also we need to map it to a membership object. I'm missing the name of the club, the start year, etc.
You can check how a membership array looks like at: https://api.fri.is/athletes/7993

Reply:
oh sorry it is my bad. you are right .we changed to  res.body[0].club.